app/tx/[signature]/page-client.tsx

```diff
@@ -1,34 +1,33 @@
 'use client';
 
-import { Address } from '@components/common/Address';
-import { BalanceDelta } from '@components/common/BalanceDelta';
-import { ErrorCard } from '@components/common/ErrorCard';
-import { InfoTooltip } from '@components/common/InfoTooltip';
-import { LoadingCard } from '@components/common/LoadingCard';
-import { Signature } from '@components/common/Signature';
-import { Slot } from '@components/common/Slot';
-import { SolBalance } from '@components/common/SolBalance';
-import { TableCardBody } from '@components/common/TableCardBody';
-import { SignatureContext } from '@components/instruction/SignatureContext';
-import { InstructionsSection } from '@components/transaction/InstructionsSection';
-import { ProgramLogSection } from '@components/transaction/ProgramLogSection';
-import { TokenBalancesCard } from '@components/transaction/TokenBalancesCard';
-import { FetchStatus } from '@providers/cache';
-import { useCluster } from '@providers/cluster';
-import { useFetchTransactionStatus, useTransactionDetails, useTransactionStatus } from '@providers/transactions';
-import { useFetchTransactionDetails } from '@providers/transactions/parsed';
-import { SystemInstruction, SystemProgram, TransactionSignature } from '@solana/web3.js';
-import { ClusterStatus } from '@utils/cluster';
-import { displayTimestamp } from '@utils/date';
-import { SignatureProps } from '@utils/index';
-import { getTransactionInstructionError } from '@utils/program-err';
-import { intoTransactionInstruction } from '@utils/tx';
-import { useClusterPath } from '@utils/url';
-import { BigNumber } from 'bignumber.js';
+import {Address} from '@components/common/Address';
+import {BalanceDelta} from '@components/common/BalanceDelta';
+import {ErrorCard} from '@components/common/ErrorCard';
+import {InfoTooltip} from '@components/common/InfoTooltip';
+import {LoadingCard} from '@components/common/LoadingCard';
+import {Signature} from '@components/common/Signature';
+import {Slot} from '@components/common/Slot';
+import {SolBalance} from '@components/common/SolBalance';
+import {TableCardBody} from '@components/common/TableCardBody';
+import {SignatureContext} from '@components/instruction/SignatureContext';
+import {InstructionsSection} from '@components/transaction/InstructionsSection';
+import {ProgramLogSection} from '@components/transaction/ProgramLogSection';
+import {TokenBalancesCard} from '@components/transaction/TokenBalancesCard';
+import {FetchStatus} from '@providers/cache';
+import {useCluster} from '@providers/cluster';
+import {useFetchTransactionStatus, useTransactionDetails, useTransactionStatus} from '@providers/transactions';
+import {useFetchTransactionDetails} from '@providers/transactions/parsed';
+import {SystemInstruction, SystemProgram, TransactionSignature} from '@solana/web3.js';
+import {Cluster, ClusterStatus} from '@utils/cluster';
+import {displayTimestamp} from '@utils/date';
+import {SignatureProps} from '@utils/index';
```

Comment:
Can you run `yarn fmt` on your code & fix these no-op changes?

These changes do nothing to affect code, but affect readability of the PR

Reply:
Sorry, fixed the format. I don't know why my IDE is doing that (fixed by using vscode now)